MagneticField/ParametrizedEngine/interface/ParabolicParametrizedMagneticField.h

```diff
@@ -5,11 +5,16 @@
 #ifndef MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
 #define MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
 
+#include <Eigen/Core>
+#include <alpaka/alpaka.hpp>
+
 namespace magneticFieldParabolicPortable {
 
   struct Parameters {
     // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
     // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+    // Hard-wired numbers defining the surfaces on which the crystal front faces lie
+
```

Comment:
I wonder where this line originates from. Maybe an earlier version of https://github.com/cms-sw/cmssw/pull/44360?  I'd remove these two lines from this PR (unless there is a strong reason to keep)
```suggestion
```

Reply:
This is an artifact from Charis's code, removed